webapp/app/js/directives/clonejobs.js

```diff
@@ -569,33 +569,33 @@ treeherder.directive('thCloneJobs', [
         // Note: I was using
         //     jobEl.style.display = "inline";
         //     jobEl.className += " filter-shown";
-        // but that didn't work reliably with the jquery selectors
+        // but the classname setting didn't work reliably with the jquery selectors
         // when it came to hiding/showing platforms and groups.  Jquery
         // couldn't detect that I'd added or removed ``filter-shown`` in
         // all cases.  So, while this is a bit slower, it's reliable.
+        //
+        // It would be great to be able to do this without adding/removing a class
         if (show) {
-            job.css("display", "inline-block");
-            job.addClass("filter-shown");
+            job[0].classList.add("filter-shown");
```

Comment:
adding the class this way was WAY faster than use ``.addClass``.  I test it on Chrome and Safari and it works great.